src/Traits/EwalletTrait.php

```diff
@@ -27,41 +30,54 @@ trait EwalletTrait
      * @Groups({"create"})
      *
      * @var string|null
+     *
+     * @Assert\Type(type="string")
      */
     protected $currency;
 
     /**
      * Ewallet id.
      *
      * @var string|null
+     *
+     * @Assert\Type(type="string")
      */
     protected $id;
 
     /**
      * Ewallet pan.
      *
      * @var string|null
+     *
+     * @Assert\Type(type="string")
      */
     protected $pan;
 
     /**
      * If is primary.
      *
      * @var bool
+     *
```

Comment:
`@Assert\NotBlank()` should be here according to the `@var` value

Reply:
Good catch. It _can_ be null though, since we're not required to actually put in this value when parsing JSON

Reply:
Does that mean `@var bool` should become `@var bool|null`?

Reply:
the latter, fixed